queries.sql

```diff
@@ -21,4 +21,15 @@ SELECT animals.name FROM animals JOIN species ON animals.species_id = species.id
 SELECT species.name, COUNT(animals.id) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Jennifer Orwell' AND species_id = 2;
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Wincheser' AND escape_attempts = 0;
-SELECT owners.full_name, MAX(animals.id) FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name, animals.id ORDER BY animals.id DESC LIMIT 1;
\ No newline at end of file
+SELECT owners.full_name, MAX(animals.id) FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name, animals.id ORDER BY animals.id DESC LIMIT 1;
+
+SELECT visits.animals_name, vets.id, MAX(visits.date_of_visit) FROM visits JOIN vets ON vets.id = visits.vets_id WHERE vets.name = 'William Thatcher' GROUP BY visits.animals_name, vets.id, visits.date_of_visit ORDER BY visits.date_of_visit DESC LIMIT 1;
+SELECT vets.name, COUNT(DISTINCT visits.animals_name) FROM vets JOIN visits ON visits.vets_id = vets.id WHERE vets.name = 'Stephanie Mendez' AND vets_id = 3 GROUP BY vets.name;
+SELECT vets.name, specializations.vets_id, specializations.species_id FROM vets JOIN specializations ON specializations.vets_id = vets.id GROUP BY vets.name, specializations.vets_id, specializations.species_id;
+SELECT visits.animals_name FROM vets JOIN visits ON visits.vets_id = vets.id WHERE vets.name = 'Stephanie Mendez' AND vets_id = 3 AND visits.date_of_visit BETWEEN '2020-04-01' and '2020-08-30' GROUP BY visits.animals_name;
+SELECT visits.animals_name, COUNT(visits.animals_name) FROM visits GROUP BY visits.animals_id ORDER BY `value_occurrence` DESC LIMIT 1;
```

Comment:
### Optional
- [ ] It would be better if you fix this query and what it does: `SELECT visits.animals_name, COUNT(visits.animals_name) FROM visits GROUP BY visits.animals_id ORDER BY "value_occurrence" DESC LIMIT 1;`

Reply:
Thank you for the suggestion. The query is not supposed to be there and so I deleted it